### PR TITLE
Schedule C endpoint

### PR DESCRIPTION
**Pull request by jontours**
Initial rough version, verifying and adding columns, filters, and sorting.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
Great start

**Comment by jontours** [comment]
This just needs documentation added for the swagger-ui and should be ready for merge.

**Comment by LindsayYoung** [comment]
I am having an issue with this example:
http://localhost:5000/v1/schedules/schedule_c/2080920101128213800/

File: webservices/resources/sched_c.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    def index_column(self):
+        return self.model.sub_id
+
+    filter_multi_fields = [
+        ('image_number', models.ScheduleC.image_number),
+        ('committee_id', models.ScheduleC.committee_id),
+    ]
+
+    filter_fulltext_fields = [
+        ('loaner_name', models.ScheduleC.loan_source_name),
```
Comment (LindsayYoung) [inline webservices/resources/sched_c.py]: We are storing this as an string but for name search we need a tsvector

Comment (jontours) [inline webservices/resources/sched_c.py]: @LindsayYoung adding a `TSVECTOR` for this columns won't be a simple thing to do, not how it's currently implemented.  For the other itemized tables we map to the `ofec...` tables, and in those tables we create a `TSVECTOR` column for fast full-text searching.

So to add that support for this we'd have to add a new TSVECTOR column and keep it update with a trigger, as documented in this reference: https://blog.lateral.io/2015/05/full-text-search-in-milliseconds-with-postgresql/

Not a hard thing to do at all, but won't be able to get in this release, so I'll just pull that filter for now and we can get it in for the release possible.

File: webservices/docs.py [hunk 2]
```diff
@@ -399,11 +399,12 @@
...
 
 SCHEDULE_C_TAG = '''
-Schdule C tag stuff
+Schedule C shows all loans, endorsements and loan guarantees a committee
+receives or makes.
 '''
 
 SCHEDULE_C = SCHEDULE_C_TAG + '''
-Schedule C stuff
+The committee continues to report the loan until it is repaid.
```
Comment (jontours) [inline webservices/docs.py]: @LindsayYoung Spelling looks good to go!!

**Comment by LindsayYoung** [comment]
Thanks for all the updates to it! looks good

**Pull request merged**